### ISSUE TITLE
Mind boggling

### ISSUE THREAD

**Issue by vmadman** [docstrap/docstrap#56]
Ok, I struggled with this for two days before deciding to open an issue here.

I know I'm probably missing a fundamental... but.... I've noticed a few weird things and I cannot wrap my head around them.  I'm confused because the project does seem to be active, with a commit only 2 days ago.. so, I assume its working for everyone else.. which is weird because I really don't see how it could _possibly_ be working for everyone else.

---

Ok, first of all... running `bower install` will, for some reason, place its output in `./public/bower`, even though `./styles/main.less` references bootstrap in `./bower_components`.  This in itself is a little puzzling because I do not see a .bowerrc file.  I assume, maybe, Docstrap was built for/with an older version of bower which had a default target of `./bower_components` and some update has moved it to `./public/bower`.

I suppose it could also be the inverse, and `./public/bower` is the old target, but my copy of bower was installed only a few days ago (via the Yeoman package) .. so, I don't know, that's just weird.

But, we can put that aside... since I can forcefully move and copy directories...

---

Second, the `bower.json` file targets bootstrap ~2.3.1, and so that is downloaded.  However, the bootswatch LESS files are clearly expecting Bootstrap 3.2.0.

I'm new to all of this, so I'm just doing a lot of guessing... but... it seems like Bootstrap 2.x named it's variables with camel case, like `@fontSize` and Bootstrap 3.x names it's variables with dashes, like `@font-size`.

Thus, _none_ of the Bootswatch 3.2  themes properly override the Bootstrap 2.3 variables and the output from `grunt less:dev` (and all other `less` tasks) will result in, basically, an unaltered Bootstrap 2.3.

---

So, I also noticed that the method that pulls the Bootswatch list targets `/` at `api.bootswatch.com` (in `Gruntfile.js`).  Earlier in my tests I realized that Bootswatch also offers a version `/3/` endpoint.  I suppose then that they also offer a `/2/` endpoint, which Docstrap should probably be targeting since it wants Bootstrap 2.3 themes.

Ok, makes sense, right?  Well... no... because I noticed that the currently targeted Bootswatch endpoint of `/` does not include the `Yeti` theme.  I took that to mean that `Yeti` was added for or around Bootstrap 3.x, since the `/3/` endpoint does return it.

In other words, the `/` endpoint for `api.bootswatch.com` returns only the Bootstrap 2 compatible themes but it returns them using the Bootstrap 3 variables.

---

Am I just being crazy here?
